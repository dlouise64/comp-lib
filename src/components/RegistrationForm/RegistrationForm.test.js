import React from 'react';
import renderer from 'react-test-renderer';
import RegistrationForm from './RegistrationForm';

test('<RegistrationForm /> renders correctly', () => {
  const tree = renderer.create(<RegistrationForm
    onSubmit={()=>{}}
  />).toJSON();
  expect(tree).toMatchSnapshot();
});
