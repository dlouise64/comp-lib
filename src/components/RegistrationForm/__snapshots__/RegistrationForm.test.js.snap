// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegistrationForm /> renders correctly 1`] = `
<div
  className="registrationForm"
>
  <p>
    <div
      style={
        Object {
          "display": "inline-block",
          "marginBottom": 16,
          "position": "relative",
        }
      }
    >
      <label
        htmlFor="registration-form-email"
        style={
          Object {
            "display": "block",
          }
        }
      >
        Email
         
        <span
          style={
            Object {
              "color": "red",
            }
          }
        >
           *
        </span>
      </label>
      <input
        className="textInput"
        id="registration-form-email"
        name="email"
        onChange={[Function]}
        type="text"
        value=""
      />
    </div>
  </p>
  <p>
    <div
      style={
        Object {
          "display": "inline-block",
          "marginBottom": 16,
          "position": "relative",
        }
      }
    >
      <label
        htmlFor="registration-form-password"
        style={
          Object {
            "display": "block",
          }
        }
      >
        Password
         
        <span
          style={
            Object {
              "color": "red",
            }
          }
        >
           *
        </span>
      </label>
      <input
        className="textInput"
        id="registration-form-password"
        maxLength={50}
        name="password"
        onChange={[Function]}
        type="password"
        value=""
      />
      <span
        onClick={[Function]}
        style={
          Object {
            "bottom": 0,
            "position": "absolute",
            "right": "-35px",
            "top": "27px",
          }
        }
      >
        <svg
          height="24"
          viewBox="0 0 22 22"
          width="24"
          xmlns="http://www.w3.org/2000/svg"
        >
          <g
            fill="#4d4d4d"
            transform="matrix(.02146 0 0 .02146 1 1)"
          >
            <path
              d="m466.07 161.53c-205.6 0-382.8 121.2-464.2 296.1-2.5 5.3-2.5 11.5 0 16.9 81.4 174.9 258.6 296.1 464.2 296.1 205.6 0 382.8-121.2 464.2-296.1 2.5-5.3 2.5-11.5 0-16.9-81.4-174.9-258.6-296.1-464.2-296.1m0 514.7c-116.1 0-210.1-94.1-210.1-210.1 0-116.1 94.1-210.1 210.1-210.1 116.1 0 210.1 94.1 210.1 210.1 0 116-94.1 210.1-210.1 210.1"
            />
            <circle
              cx="466.08"
              cy="466.02"
              r="134.5"
            />
          </g>
        </svg>
      </span>
    </div>
  </p>
  <p>
    <input
      onClick={[Function]}
      type="submit"
      value="Register"
    />
  </p>
</div>
`;
